tests/test_cli_compile.py

```diff
@@ -612,38 +630,31 @@ def test_local_url_package(
     assert dependency in out.stderr
 
 
-@pytest.mark.parametrize(
-    ("line", "dependency", "rewritten_line"),
-    (
-        pytest.param(
-            os.path.join(
-                MINIMAL_WHEELS_PATH, "small_fake_with_deps-0.1-py2.py3-none-any.whl"
-            ),
-            "\nfile:small_fake_with_deps-0.1-py2.py3-none-any.whl",
```

Comment:
This test is now a standard unit test called `test_relative_file_uri_package` (just below).